src/styles/card_style.scss

```diff
@@ -0,0 +1,88 @@
+.card {
```

Comment:
Rename this file to just card if it is inside the styles folder so it`s redundant to name it like this